phantomgames/Search/SearchGameViewController.swift

```diff
@@ -28,41 +31,56 @@ class SearchGameViewController: UIViewController {
         tableView.delegate = self
         tableView.dataSource = self
     }
+    private func setupSearchController () {
+        searchController.searchResultsUpdater = self
+        searchController.obscuresBackgroundDuringPresentation = false
+        searchController.hidesNavigationBarDuringPresentation = false
+        searchController.searchBar.placeholder = Constants.SearchConstants.searchBarPlaceholder
+        navigationItem.searchController = searchController
+        definesPresentationContext = false
+        navigationItem.hidesSearchBarWhenScrolling = false
+    }
 }
 
-// MARK: - TableView Delegate
+// MARK:  Search Controller Functions
+
+extension SearchGameViewController: UISearchResultsUpdating {
+    
+    func updateSearchResults(for searchController: UISearchController)
+    { viewModel.updateSearchController(searchBarText: searchController.searchBar.text) }
```

Comment:
please put the opening brace next to the function 
 func updateSearchResults(for searchController: UISearchController) {
 //code
}